src/main/webapp/WEB-INF/pages/manage/overview_metadata-bootstrap.ftl

```diff
@@ -52,6 +52,18 @@
                         <@s.submit cssClass="btn btn-sm btn-outline-success" name="edit" key="button.edit"/>
```

Comment:
please add a css class 'my-1' here

Reply:
I'm a bit lost... Did I change something there? :) 
I guess it's because I did code change after your comment? So lines are mixed up...

If you mean putting the emlReplace/emlClear buttons in a group, I agree with you, but I did a copy of https://github.com/gbif/ipt/blob/bootstrap-ui/src/main/webapp/WEB-INF/pages/manage/overview_data-bootstrap.ftl#L69 where there is no button group.